migrations/2023-06-21-153242_add_captcha/up.sql

```diff
@@ -0,0 +1,5 @@
+create table user_captcha(
```

Comment:
I would simply call this `captcha` in case we want to reuse it later for post creation or other cases.

Reply:
i ended up having to re-generate the diesel diff file since this squeezes the new table right above the comment table